templates/DO-178C/doc/software/requirements/SDD.sdoc

```diff
@@ -4,3 +4,149 @@ TITLE: Software Design Description
 [FREETEXT]
 Defines the software architecture and low level requirements, that will satisfy high level requirements.
 [/FREETEXT]
+
+[SECTION]
+TITLE: Overview
+
+[FREETEXT]
+Detailed description of how the software requirements are allocated to processors and tasks
+[/FREETEXT]
+
+[/SECTION]
+
+[SECTION]
+TITLE: Architecture
+
+[FREETEXT]
+Defines the software structure to achieve the high level requirements
+[/FREETEXT]
+
+[/SECTION]
+
+[SECTION]
+TITLE: Interface
+
+[FREETEXT]
+Describes input and output interfaces
+[/FREETEXT]
+
+[REQUIREMENT]
+UID: REQ-A429
+TITLE: Arinc 429 label format
+STATEMENT: >>>
+The IMU_AZ data is read from the data field in the Arinc word defined by:
+
+.. image:: ../../../_assets/A429.*
+<<<
+COMMENT: >>>
+https://fr.wikipedia.org/wiki/ARINC_429
+<<<
+
+[SECTION]
+TITLE: Low level requirements
+
+[REQUIREMENT]
+UID: LLR-001
+REFS:
+- TYPE: Parent
+  VALUE: HLR-001
+TITLE: LLR title example
+STATEMENT: >>>
+The software component **imu** shall ....
+<<<
+
+[/SECTION]
+
+[SECTION]
+TITLE: Detailed design
+
+[FREETEXT]
+.. doxygenfile:: imu.h
```

Comment:
As mentioned in the email today, please remove the `doxygenfile` from this patch for now. At my next opportunity, I will fix StrictDoc and it will resolve this directive to "empty string" when running HTML export.